src/seedu/addressbook/ui/TextUi.java

```diff
@@ -13,15 +10,14 @@
 import java.util.Optional;
 import java.util.Scanner;
 
-import seedu.addressbook.commands.CommandResult;
-import seedu.addressbook.data.person.ReadOnlyPerson;
+import static seedu.addressbook.common.Messages.*;
```

Comment:
Imported classes should always be stated explicitly! If your IDE is automatically converting this, please do change the settings to prevent it from doing so.